language-plutus-core/bench/Bench.hs

```diff
@@ -3,10 +3,20 @@ module Main (main) where
 import           Codec.Serialise
 import           Control.Monad
 import           Criterion.Main
-import qualified Data.ByteString.Lazy       as BSL
+import qualified Data.ByteString.Lazy                 as BSL
+import qualified Data.Map                             as M
 import           Language.PlutusCore
+import           Language.PlutusCore.Constant
+import           Language.PlutusCore.Constant.Dynamic
 import           Language.PlutusCore.Pretty
 
+traceBuiltins :: QuoteT (Either (Error ())) DynamicBuiltinNameTypes
+traceBuiltins = dynamicBuiltinNameMeaningsToTypes () $ DynamicBuiltinNameMeanings $ M.fromList
+    [ (dynamicCharToStringName, dynamicCharToStringMeaning)
+    , (dynamicTraceName, dynamicTraceMeaning)
+    , (dynamicAppendName, dynamicAppendMeaning)
+    ]
+
 main :: IO ()
 main =
     defaultMain [ env largeTypeFiles $ \ ~(f, g, h) ->
```

Comment:
Those large files are still not large at all, right? @michaelpj has plenty of very good test cases in `plutus-tx` and we also have some real and actually used contracts. We definitely should benchmarks those ones.

Reply:
But we can do that in a separate PR.

Reply:
That is correct. It would be wise to add *every* example, which I would prefer to do after this. The main value in keeping the old test cases is that they do not require dynamic builtins (and more importantly, they show us how things scale)